molecule/driver/docker.py

```diff
@@ -111,11 +111,21 @@ class Docker(Driver):
             buildargs:
                 http_proxy: http://proxy.example.com:8080/
 
-    If specifying the `CMD`_ directive in your ``Dockerfile.j2`` or consuming a
-    built image which declares a ``CMD`` directive, then you must set
-    ``override_command: False``. Otherwise, Molecule takes care to honour the
-    value of the ``command`` key or uses the default of ``bash -c "while true;
-    do sleep 10000; done"`` to run the container until it is provisioned.
+    Molecule takes care to honour the value of the ``entrypoint`` and
```

Comment:
Just a quick question; do we use British or US English in docs? ('honour' vs 'honor')

Reply:
I just try to rewrite the comment based on original sentence structure; IMHO, vote for US English ;-)